tests/meltano/core/tracking/test_exception.py

```diff
@@ -97,6 +99,10 @@ def test_simple_exception_context():
         assert key in tb_data[0]
 
 
+@pytest.mark.skipif(
```

Comment:
This is pretty important as it's an example of a new Test was was written that doesn't pass on Windows. I went ahead and ignored this test. We'll probably want to at a minimum add to a guide that if a pass doesn't test on Windows you'll want to "ignore" the test, although that sounds terrible at least it won't impact development velocity as badly.

Punting these questions to https://github.com/meltano/meltano/issues/3444 is ok, but it does mean that we are currently telling everyone to just ignore all the Windows failures until we have a better solution here https://github.com/meltano/meltano/issues/3444 which will have to be some form of either always ignore the error, fix the error, or bring someone in to help fix the error.

Reply:
After chatting with @aaronsteers  it sounds like this is fine. I'll leave this open and let @aaronsteers  resolve just in case :D

Reply:
I believe I've fixed the Windows incompatibility in my branch for Python 3.10 support. I expect this PR will be merged first, so it'll be easy for me to verify that.

Reply:
@visch and @WillDaSilva - Resolving this. Yes, makes sense. 👍